org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/ast/AllocationExpression.java

```diff
@@ -541,9 +541,10 @@ public TypeBinding resolveType(BlockScope scope) {
 
 protected boolean isMissingTypeRelevant() {
 	if (this.binding != null && this.binding.isVarargs()) {
-		if (this.arguments.length < this.binding.parameters.length) {
+		int argLen = this.arguments != null ? this.arguments.length : 0;
+		if (0 < this.binding.parameters.length) {
```

Comment:
Should this use argLen instead of 0 in the condition?

Reply:
> Should this use argLen instead of 0 in the condition?

OMG, did I write that? Thanks, eagle-eyed @szarnekow